src/test/resources/network/brightspots/rcv/test_data/minneapolis_multi_seat_threshold/minneapolis_multi_seat_threshold_config.json

```diff
@@ -11,7 +11,7 @@
   },
   "cvrFileSources": [
     {
-    "filePath" : "minneapolis_multi_seat_threshold_cvr.xlsx",
+    "filePath" : "../_shared/minneapolis_multi_seat_cvr.xlsx",
```

Comment:
It seems like this one might not actually be shared between any other tests? Are we pre-emptively moving CVRs into the `_shared` folder, or was this a mistake?